app/views/users/sessions.py

```diff
@@ -141,6 +144,93 @@ def add_speaker_view(event_id, session_id):
         return redirect(url_for('.index_view', event_id=event_id))
 
 
+@event_sessions.route('/download_speakers_as_csv/')
+@belongs_to_event
+@can_access
+def download_speakers_as_csv(event_id):
+    sessions = DataGetter.get_sessions_by_event_id(event_id)
+    event = DataGetter.get_event(event_id)
+    data = "Speaker Name, Speaker Session(s) , Speaker Email, Speaker Status," \
+           " Speaker Mobile, Speaker Organisation, Speaker Position \n"
+
+    for session in sessions:
+        for speaker in session.speakers:
+            if speaker.name:
```

Comment:
This could be re-written as 
`data += (speaker.name + ',') if speaker.name else ','`